src/Medology/Behat/CsvContext.php

```diff
@@ -118,4 +119,28 @@ private function assertNoAdditionalHeaders(array $required, array $actual, strin
             throw new Exception("CSV '$csvName' contains extra headers '" . implode("', '", $diff) . "'");
         }
     }
+
+    /**
+     * Update expected data injecting model values.
```

Comment:
When you move this method, please also update this description:

```suggestion
     * Parses each TableNode cell for references to stored items and replaces them with the value from the store.
```